### DISCUSSION TITLE
How to disable automatic directory creation?

### DISCUSSION THREAD

**Discussion by olegans1972** [seaweedfs/seaweedfs#4275] [General]
At the documentation I see  https://github.com/seaweedfs/seaweedfs/wiki/Filer-Server-API the directories /path/to will be automatically created. 

Is there an option to disable this feature. I want to have a set of certain directories set, for example, in some config, so that others cannot be made without first adding them to this config, for example

**Comment by chrislusf** [comment]
not yet. Worth a PR.
